scripts/register_oauth_scopes.py

```diff
@@ -27,43 +26,38 @@ def get_or_create(name, description, save=True):
     """
     Populate or update the database entry, as needed
 
-    :param name:
-    :param description:
-    :return:
+    :param name: the name of the scope
+    :param description: the description of the scope
+    :return: the scope object
     """
+
     if name != name.lower():
         raise ValueError('Scope names are case-sensitive, and should always be lower-case.')
 
     try:
-        scope_obj = ApiOAuth2Scope.find_one(Q('name', 'eq', name))
-    except NoResultsFound:
+        scope_obj = ApiOAuth2Scope.objects.get(name=name)
+        setattr(scope_obj, 'description', description)
+        print("Updating existing database entry for: %s", name)
+    except ApiOAuth2Scope.DoesNotExist:
         scope_obj = ApiOAuth2Scope(name=name, description=description)
-        print "Created new database entry for: ", name
-    else:
-        scope_obj.description = description
-        print "Updating existing database entry for: ", name
+        print("Created new database entry for: %s", name)
 
-    if save is True:
+    if save:
         scope_obj.save()
-    return scope_obj
-
 
-def set_backend():
-    """Ensure a storage backend is set up for this model"""
-    set_up_storage([ApiOAuth2Scope], storage.MongoStorage)
+    return scope_obj
 
 
-def do_populate():
+def do_populate(clear=False):
     """
-    :param dict scope_dict: Given a dictionary of scope definitions, {name: scope_namedtuple}, load the
-        resulting data into a database collection
-    :return:
+    Given a dictionary of scope definitions, {name: scope_namedtuple}, load the
+    resulting data into a database collection
     """
+
     scope_dict = oauth_scopes.public_scopes
 
-    # Clear the scope collection and populate w/ only public scopes,
-    # nothing references these objects other than CAS in name only.
-    ApiOAuth2Scope.remove()
+    if clear:
```

Comment:
The previous script clears the collection every time the scripts is run, which is preserved here as an option.